utils_cv/common/gpu.py

```diff
@@ -51,11 +51,7 @@ def torch_device():
 
 def num_devices():
     """ Gets the number of devices based on cpu/gpu """
-    return (
-        torch.cuda.device_count()
-        if torch.cuda.is_available()
-        else 1
-    )
+    return torch.cuda.device_count() if torch.cuda.is_available() else 1
```

Comment:
General comment: before this goes into staging, could you please add a readme.md file which points out that this is work in progress. Doesn't need any other text, see e.g.
https://github.com/microsoft/computervision-recipes/blob/master/scenarios/action_recognition/README.md